chain/wallet/key/key.go

```diff
@@ -50,6 +52,17 @@ func NewKey(keyinfo types.KeyInfo) (*Key, error) {
 		if err != nil {
 			return nil, xerrors.Errorf("converting Secp256k1 to address: %w", err)
 		}
+	case types.KTDelegated:
+		// Assume eth for now
```

Comment:
Can we add some standard NOTE comment or something like that to places where we are making assumptions like this?

Reply:
Can do, referencing the transitory Delegated signature verification as defined in FIP-0055.